File: traslate_py_C/example6.c
#include <stdio.h>
#include <math.h>

int main(void){
    
    int opcion, diametro, area, perimetro;
    float radio, pi;
    opcion = 0;
    pi = 3.14;
    
    while(opcion != 4){
        
        printf("Escoge una opcion: ");
        printf("1) Calcular el diametro.");
        printf("2) Calcular el perimetro.");
        printf("3) Calcular el area.");
        printf("4) Salir.");
        printf("Teclea 1, 2, 3, o 4 y pulsa el retorno de carro: \n");
        scanf("%d", &opcion);
        printf("Dame el radio de un circulo: ");
        scanf("%f", &radio);
        
        if(opcion == 1){
            diametro = 2 * radio;
            printf("El diámetro es %d", diametro);
        } else if(opcion == 2){
            perimetro = 2 * pi * radio;
            printf("El perimetro es %d", perimetro);
        } else if(opcion == 3){
            area = pi * radio * 8;
            printf("El  area es %d", area);
        } else if(opcion < 0){
            printf("S ́lo hay cuatro opciones: 1, 2, 3 o 4. T ́has tecleado %d", opcion);
        } else if(opcion > 4){
            printf("S ́lo hay cuatro opciones: 1, 2, 3 o 4. T ́has tecleado %d", opcion);
        }
        
    }
    
    return 0;
}

File: traslate_py_C/example6.py
from math import pi

opcion = 0
while opcion != 4:
    print "Escoge una opcion: "
    print "1) Calcular el diametro."
    print "2) Calcular el perimetro."
    print "3) Calcular el area."
    print "4) Salir."
    
    opcion = int(raw_input("Teclea 1, 2, 3 o 4 y pulsa el retorno de carro: "))
    radio = float(raw_input("Dame el radio de un circulo: "))

    if opcion == 1:
        diametro = 2 * radio
        print "El diámetro es", diametro
    elif opcion == 2:
        perimetro = 2 * pi * radio
        print "El perimetro es", perimetro
    elif opcion == 3:
        area = pi * radio ** 2
        print "El  area es", area
    elif opcion < 0 or opcion > 4:
        print "S ́lo hay cuatro opciones: 1, 2, 3 o 4. T ́has tecleado", opcion
